src/_posts/languages/python/django/2000-01-01-start.md

```diff
@@ -100,6 +100,18 @@ Then update `Procfile`:
 web: bash bin/start.sh
 ```
 
+### Using Daphne
+
+It is also possible to use Daphne as an ASGI server.
+
+Here an example start.sh script to use Daphne:
```

Comment:
issue: 
- the sentence is not really well written. It's really hard to understand. I don't think it's a very good English.
- it's kind of hard to understand you refer to the `start.sh` script from the previous section
- I am not sure it should be a different paragraph than the previous one.
question: How is it possible to install the `daphne` CLI?

```suggestion
It is also possible to use Daphne as an ASGI server. You can update the `bin/start.sh` script described in the previous section to use Daphne to start the application:
```